source/frontend/src/features/leases/detail/LeasePages/details/PropertyInformation.tsx

```diff
@@ -57,10 +44,7 @@ export const PropertyInformation: React.FunctionComponent<
       </SectionField>
       {!hideAddress ? (
         <SectionField label="Address" labelWidth="3" className="py-2">
-          <AddressSubForm
-            nameSpace={withNameSpace(nameSpace, 'property.address')}
-            disabled={disabled}
-          />
+          <>{property.property.address.streetAddress3}</>
```

Comment:
This doesn't seem right, the form used to display the address subform with a number of address fields but now we only display streetAddress3?

Reply:
Good catch, I was testing the address and stashed the view. I've updated it now to use the new address view